src/Turbo/doc/index.rst

```diff
@@ -288,6 +288,42 @@ With content:
         A placeholder.
     </turbo-frame>
 
+Minimal layout for Turbo Frames
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.22
+
+    The minimal layout for Turbo Frames was added in Turbo 2.22.
+
+Since Turbo does not need the content outside of the frame, reducing the amount that is rendered can be a useful optimisation. However, this optimisation prevents responses from specifying ``head`` content as well. There are cases where it would be useful for Turbo to have access to items specified in ``head``. To specify the heads, you must then use a minimal layout for frame, rather than no layout. With this, applications can render content into the ``head`` if they want.
```

Comment:
> reducing the amount that is rendered can be a useful optimisation

I'd present first an example of this (with controller code and renderblock for instance)

> However, this optimisation prevents responses from specifying

This part can be removed i think, as the same idea is on the next sentence. And i feel it's a bit fearfull and would not want to alarm users if they dot need any meta tags (which is, as i see it, the most standard case, no ?)

> With this, applications can render content into the ``head`` if they want.

Maybe something like "it allows you to set meta tags while still having a minimal ...." ?